### DISCUSSION TITLE
Migration from 16 to 17 possible ?

### DISCUSSION THREAD

**Discussion by fmasson-arkod** [OCA/OpenUpgrade#4332] [Q&A] [answered]
Hello everyone, 

I wonder if its yet possible or not to migrate from 16 to 17 using OCA OpenUpgrate script and framework for version 17. As i successfully do for the version 15 to 16. 

Apparently the 17 branch is a bit late according to the 16 (450+ commits behind) and also cannot find the openupgrade framework in the repo.

regards.

**Comment by rvalyi** [comment, marked as answer]
Hello, it is barely started at the framework level, so expect a decent coverage only in some 10 months or so. May be just a few months if migrating only the base and product modules would do it for you. If you need things to happen faster you may ask to the main contributors: Tecnativa, Forgeflow, Sylvain Legal, Stefan Rijnhart, Copiteasy, Le Filament etc...

**Reply by fmasson-arkod** [reply]
Thank you Raphaël !
